app/models/concerns/enc_output.rb

```diff
@@ -43,6 +43,7 @@ module EncOutput
   def to_enc
     own_attributes = {}
     self.attributes.each do |k, v|
+      v = send(k) if v.is_a? String and v.include? Encryptable::ENCRYPTION_PREFIX
```

Comment:
`v.starts_with?` would be better I think.
Also, use `&&` not `and` (http://devblog.avdi.org/2010/08/02/using-and-and-or-in-ruby/).